shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -860,6 +862,7 @@ void onLowMemory() {
     ensureAlive();
     flutterEngine.getDartExecutor().notifyLowMemoryWarning();
     flutterEngine.getSystemChannel().sendMemoryPressureWarning();
+    flutterEngine.getRenderer().onTrimMemory(TRIM_MEMORY_COMPLETE);
```

Comment:
weird. there's no call to `delegate.onLowMemory` from `io.flutter.embedding.android.FlutterActivity`, but there's one from `io.flutter.embedding.android.FlutterFragment`. This seems wrong. Do we know why that's the case? cc @dnfield

Reply:
Activity is using `onTrimMemory`.

Reply:
According to the Android documentation, `onLowMemory` is only used for API levels lower than 14:
>To support API levels lower than 14, you can use the [ComponentCallbacks.onLowMemory()](https://developer.android.com/reference/android/content/ComponentCallbacks#onLowMemory()) method as a fallback that's roughly equivalent to the [ComponentCallbacks2#TRIM_MEMORY_COMPLETE](https://developer.android.com/reference/android/content/ComponentCallbacks2#TRIM_MEMORY_COMPLETE) level.

Reply:
It's confusing that `onLowMemory` is called from  `FlutterFragment` but not `FlutterActivity`.  We don't support API level 14, so should we remove all instances of `onLowMemory`?

Reply:
> It's confusing that `onLowMemory` is called from `FlutterFragment` but not `FlutterActivity`. We don't support API level 14, so should we remove all instances of `onLowMemory`?

done.